diff --git a/dist/ssg.js b/dist/ssg.js
index eec072669e5441fef9b0456d2156abfc66b0ce09..0000000000000000000000000000000000000000 100644
--- a/dist/ssg.js
+++ b/dist/ssg.js
@@ -44,6 +44,8 @@ const ssgBuild = (options) => {
         }
       }
       const server = await createServer({
+        root: config.root,
+        define: config.define,
         resolve: {
           ...config.resolve,
           builtins: [...config.resolve.builtins, /^node:/]
